### PR TITLE
docs: adding more description to the tasks

### PR DESCRIPTION
**Pull request by gouravkrosx**
Signed-off-by: gouravkrosx <gouravgreatkr@gmail.com>

### CODE REVIEW

File: 2023/README.md [hunk 1]
```diff
@@ -57,11 +57,9 @@ _________________
 * Add support for `Get` curl command for the test case.
 * Add support to `Run` individual test case.
 * On the CLI add a vertical line separator to show differences in the test cases 
-(show complete tests info on both sides, only differences in different color)
+(show complete tests info on both sides, only highlight differences with different color)
 * Add support for commands in Keploy CLI, for eg: `show [test_id];` will 
-show that specific test case among various of testruns, `show [test_run_id];` will show the info related to it.
-* Rename test files and mock files to method_endpoint.
```
Comment (nehagup) [inline 2023/README.md:63]: why is this and the following task removed? It's a GFI that can be done within this project scope.

File: 2023/README.md [hunk 2]
```diff
@@ -78,11 +76,11 @@ show that specific test case among various of testruns, `show [test_run_id];` wi
...
 
 ### 2. Keploy Java SDK
-**Mentors**: Shubham Jain, Sarthak Shyngle, Gourav Kumar.
+**Mentors**: Sarthak Shyngle, Gourav Kumar, Charan Kamarapu.
 
 **Goals & Ideas**
-* Add support for `MongoDb` framework.
-* 
+* Add support for `MongoDB` framework.
+* Add support for `Apache Kafka`.
```
Comment (nehagup) [inline 2023/README.md:83]: We need more frameworks to add as a task. Maybe let's add some more popular ones.

File: 2023/README.md [hunk 3]
```diff
@@ -102,15 +101,20 @@ show that specific test case among various of testruns, `show [test_run_id];` wi
 **Mentors**: Charan Kamarapu, Ritik Jain, Shubham Jain.
 
 **Goals & Ideas**
-* Add support for dependencies in typescript.
-* 
+* Add support for `Httpclients` in typescript. 
+    - This sdk currently supports `node-fetch` httpClient, to support other httpclients like `axios`, `http`, etc. You can take the 1st ref below which intercepts other popular httpClients.
+* Add support for popular `Sql` databases. For eg: `PostgreSQL`, `MySQL`, etc. By adding support for `Sequelize`.
+* Add support for `MongoDB` framework.
+
```
Comment (nehagup) [inline 2023/README.md:108]: Needs more framework, task is too short for even 175 hours.

File: 2023/README.md (hunk 2 above)
Comment (slayerjain) [reply 2023/README.md:83]: We can add RabbitMQ, GCP, and Azure SDKs. Native HTTP Client

File: 2023/README.md (hunk 3 above)
Comment (slayerjain) [reply 2023/README.md:108]: Add support for Amazon S3, Dynamodb, Azure blob storage, and Google Cloud Storage using the HTTP Client. 
Add support for Redis and elastic-search.

**Review by nehagup — APPROVED** [review]
LGTM 🥳

**Pull request merged**